### PR TITLE
completed html-web-page -Andrew Owyang

### PR DESCRIPTION
**Pull request by Ajowyang**
Screenshot 2024-07-16 at 10 06 16 AM

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    <form action="submit">
+      <input type="text" value="Your Name" style="color: grey" />
+      <select name="plans" id="plans">
+        <option value="basic">Basic Plan</option>
+        <option value="deluxe">Deluxe Plan</option>
+        <option value="jellygiant">Jelly Giant Plan</option>
+      </select>
+      <button type="reset">Submit</button>
+    </form>
```
Comment (brandon-moy) [inline html-web-page/index.html]: Should this button `reset` or `submit` the form?

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      </tbody>
+    </table>
+
+    <p>Like what you see? Become a part of the JOTMC family today!</p>
+
+    <form action="submit">
+      <input type="text" value="Your Name" style="color: grey" />
+      <select name="plans" id="plans">
+        <option value="basic">Basic Plan</option>
+        <option value="deluxe">Deluxe Plan</option>
```
Comment (brandon-moy) [inline html-web-page/index.html]: What option is shown when the user first visits the page?

**Review by brandon-moy — CHANGES_REQUESTED** [review]
Looks great! Just a couple of small tweaks here.

File: html-web-page/index.html (hunk 1 above)
Comment (Ajowyang) [reply html-web-page/index.html]: submit

**Review by brandon-moy — CHANGES_REQUESTED** [review]
Still need the change for your `select` element's first `option` that shows

**Review by brandon-moy — APPROVED** [review]
Approved 👍

**Pull request merged**